implementations/README.md

```diff
@@ -72,22 +68,7 @@ The table below contains a list of [ERC165]() interface IDs used by the [Solidit
 | :------------------------------------------------------------------------------------------------------------------------ | :----------- | :------------------------------------------------------ |
 | ERC725X                                                                                                                   | `0x44c028fe` | General executor                                        |
 | ERC725Y                                                                                                                   | `0x5a988c0f` | General key-value store                                 |
-| ERC725Account                                                                                                             | `0x63cb749b` | `ERC725X` + `ERC725Y` + `LSP1` + `ERC1271`              |
-| [ERC165](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-165.md)                                                    | `0x01ffc9a7` | Standard Interface Detection for Contracts              |
-| [ERC1271](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1271.md)                                                  | `0x1626ba7e` | Standard Signature Validation Method for Contracts      |
```

Comment:
We should keep the row for `ERC165`, as `ERC725X` and `Y` implement them each individually.